lib/msal-common/src/authority/ICloudInstanceDiscoveryResponse.ts

```diff
@@ -3,9 +3,12 @@
  * Licensed under the MIT License.
  */
 
+import { ICloudDiscoveryMetadata } from "./ICloudDiscoveryMetadata";
+
 /**
  * The OpenID Configuration Endpoint Response interface. Used by the authority class to get relevant OAuth endpoints.
  */
-export interface OpenIdConfigResponse {
+export interface ICloudInstanceDiscoveryResponse {
```

Comment:
make this a type instead of an interface, take off the `I` and make it `CloudInstanceDiscoveryResponse`.